### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by SilvanVerhoeven**
- mention Python 3.7 or higher
- Add troubleshooting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +71,13 @@ via [Yarn](https://yarnpkg.com/lang/en/).
 2. go into the directory `static`
 3. run the command `yarn`
 
+#### Troubleshooting
+
+| Error         | Solution    |
+| ------------- |-------------|
+|`ERROR: Could not find a version that satisfies the requirement channels-rabbitmq==2.0.0 [...] ERROR: No matching distribution found for channels-rabbitmq==2.0.0`      | Check the python version in your environment. Are you using a python version of 3.7 or higher? |
```
Comment (janno42) [inline README.md:78]: rabbitmq will be removed in #764, so you can remove this part

File: README.md [hunk 2]
```diff
@@ -18,6 +18,8 @@ git clone https://github.com/fsr-itse/1327.git
 
 Freshly created code needs to be tested - besides our use of unit tests, linting and continous integration, it is possible to run the application in a non-production environment using *Vagrant* or a *Virtual Environment*.
 
+1327 runs on `Python 3.7` or higher.
```
Comment (Paula-Kli) [inline README.md:21]: it runs on Python 3.6 with redis which will be there again after #764

File: README.md [hunk 3]
```diff
@@ -42,10 +44,20 @@ To login with your local user instead of the default OpenID login, you have to v
 
 ### Virtual Environment
 
-Another way of executing this django application is the use of a virtual python environment. This way bypasses the needs for a virtual machine and simplifies the life with multiple python versions installed:
+Another way of executing this django application is the use of a virtual python environment. This way bypasses the needs for a virtual machine and simplifies the life with multiple python versions installed.
+
+Before creating a virtual environment, make sure to use `Python 3.7` or higher:
```
Comment (Paula-Kli) [inline README.md:49]: Python 3.6 should be possible as well

**Comment by frcroth** [comment]
Could change the link from fsr-itse to fsr-de as well

File: README.md (hunk 3 above)
Comment (SilvanVerhoeven) [reply README.md:49]: It wasn't

File: README.md (hunk 2 above)
Comment (SilvanVerhoeven) [reply README.md:21]: Aha!

**Review by janno42 — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -13,11 +13,13 @@ A student representatives website. This project is live! You can see a working i
 To be able to contribute to 1327, you need to get the source code with all dependencies. Please note, that submodules for *bootstrap*, *font-awesome* and *bootstrap-markdown* are in use:
```
Comment (jeriox) [inline README.md:13]: this is not correct anymore

Comment (SilvanVerhoeven) [reply README.md:13]: How can it be corrected? Just by removing these two sentences?

Comment (jeriox) [reply README.md:13]: yes. bootstrap/fa are collected via yarn, so following those steps is enough to set everything up

**Review by jeriox — APPROVED** [review]

**Pull request merged**